aws-greengrass-testing-platform/aws-greengrass-testing-platform-api/src/main/java/com/aws/greengrass/testing/platform/linux/LinuxNetworkUtils.java

```diff
@@ -100,4 +96,14 @@ private void modifyMqttConnection(String action) throws IOException, Interrupted
             }
         }
     }
+
+    @Override
+    public void disconnectNetwork() {
+        modifyConnection(ENABLE_OPTION, NETWORK_PORTS);
```

Comment:
why "enable option" instead of the "append option"? Should the MQTT version be using append?

Reply:
I took reference from existing internal test framework using append for mqtt and insert for network port. 
I tested with append for network port and it works. So, updating to append.